src/Components/DetailsPage/DetailsPage.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import "./DetailsPage.css";
+import logo from '../images/logo.png';
+
+const DetailsPage = ({}) => {
+  return (
+    <div className="details-container">
+      <div className="image-latin-common-container">
+        <div className="pic">
+          <img className="details-image" src={logo} alt="Logo" />
+        </div>
+        <div className="latin-common-edibility-taxonomy">
+          <div className="name-taxonomy-cont">
+            <div className="name-holder">
+              <h2>Amanita parcivolvata</h2>
+              <p>False Caesar's Mushroom, Ringless False Fly Agaric</p>
+            </div>
+            <div className="taxonomy">
+              <h3>Taxonomy:</h3>
+              <p>Genus: "Amanita",</p>
+              <p>Order: "Agraicales",</p>
+              <p>Family: "Amanitaceae",</p>
+              <p>Phylum: "Basidiomycota",</p>
+              <p> Kingdom: "Fungi"</p>
+            </div>
+          </div>
+          <div className="info">
```

Comment:
I think it will be super easy to pass down the props and get this all set up for data! Great job!!